bundle/NamedObject/Provider/Loading.php

```diff
@@ -0,0 +1,203 @@
+     * @var \eZ\Publish\Core\MVC\ConfigResolverInterface
+     */
+    private $configResolver;
+
+    /**
+     * @var array
+     */
+    private $configuration;
+
+    public function __construct(
+        LoadService $loadService,
+        ?TagsService $tagsService,
+        ConfigResolverInterface $configResolver
+    ) {
+        $this->loadService = $loadService;
+        $this->tagsService = $tagsService;
+        $this->configResolver = $configResolver;
+    }
+
+    public function hasContent(string $name): bool
+    {
+        $this->setConfiguration();
+
+        return isset($this->configuration['content'][$name]);
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @throws \Netgen\EzPlatformSiteApi\API\Exceptions\TranslationNotMatchedException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     */
+    public function getContent(string $name): Content
+    {
+        $contentId = $this->getContentId($name);
+
+        if ($contentId !== null) {
+            return $this->loadService->loadContent($contentId);
+        }
+
+        $contentRemoteId = $this->getContentRemoteId($name);
+
+        if ($contentRemoteId !== null) {
+            return $this->loadService->loadContentByRemoteId($contentRemoteId);
+        }
+
+        throw new LogicException('Named Content "' . $name . '" is not configured');
+    }
+
+    public function hasLocation(string $name): bool
+    {
+        $this->setConfiguration();
+
+        return isset($this->configuration['location'][$name]);
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @throws \Netgen\EzPlatformSiteApi\API\Exceptions\TranslationNotMatchedException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     */
+    public function getLocation(string $name): Location
+    {
+        $locationId = $this->getLocationId($name);
+
+        if ($locationId !== null) {
+            return $this->loadService->loadLocation($locationId);
+        }
+
+        $locationRemoteId = $this->getLocationRemoteId($name);
+
+        if ($locationRemoteId !== null) {
+            return $this->loadService->loadLocationByRemoteId($locationRemoteId);
+        }
+
+        throw new LogicException('Named Location "' . $name . '" is not configured');
+    }
+
+    public function hasTag(string $name): bool
+    {
+        $this->setConfiguration();
+
+        return isset($this->configuration['tag'][$name]);
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     */
+    public function getTag(string $name): Tag
+    {
+        if ($this->tagsService === null) {
+            throw new RuntimeException('Missing Netgen TagsBundle package (netgen/TagsBundle)');
```

Comment:
nitpick: lowercase `netgen/TagsBundle` :) Especially since non-lowercase package names are deprecated by Composer.

Reply:
Fixed in https://github.com/netgen/ezplatform-site-api/pull/145/commits/b4cee630226e62b70178f91040ffb4b9a2e21b0a